apparmor: Add opencl support

AppArmor in Debian Buster now has OpenCL abstractions.

Include OpenCL abstractions to fix OpenCL usage in Calc.
Index: libreoffice-6.1.4/sysui/desktop/apparmor/program.soffice.bin
===================================================================
--- libreoffice-6.1.4.orig/sysui/desktop/apparmor/program.soffice.bin	2019-01-14 19:29:25.000000000 +0200
+++ libreoffice-6.1.4/sysui/desktop/apparmor/program.soffice.bin	2019-01-16 18:43:17.051781559 +0200
@@ -98,6 +98,10 @@
 
   #include <abstractions/user-tmp>
 
+  #include <abstractions/opencl-intel>
+  #include <abstractions/opencl-mesa>
+  #include <abstractions/opencl-nvidia>
+
   #List directories for file browser
   /                                     r,
   /**/                                  r,
